.github/workflows/test-frontend.yaml

```
@@ -102,15 +102,45 @@ jobs:
           sudo curl -fLo /usr/local/lib/docker/cli-plugins/docker-compose https://github.com/docker/compose/releases/download/v2.29.1/docker-compose-linux-x86_64
           sudo chmod +x /usr/local/lib/docker/cli-plugins/docker-compose
 
-      - name: Install automation dependencies
+      - name: Install Automation Dependencies
         run: pnpm install --frozen-lockfile
         working-directory: automation
 
-      - name: Install hedera globally
-        run: npm i -g @hashgraph/hedera-local
+      - name: Setup Kind
+        uses: helm/kind-action@a1b0e391336a6ee6713a0583f8c6240d70863de3 # v1.12.0
+        with:
+          install_only: true
+          node_image: kindest/node:v1.31.4@sha256:2cb39f7295fe7eafee0842b1052a599a4fb0f8bcf3f83d96c7f4864c357c6c30
+          version: v0.26.0
+          kubectl_version: v1.31.4
+          verbosity: 3
+          wait: 120s
+
+      - name: Install Solo CLI
+        run: |
+          set -euo pipefail
+          npm install -g @hashgraph/solo
```

```suggestion
          npm install -g @hashgraph/solo@${{ vars.SOLO_VERSION }}
```

We should set solo to a specific version. In consensus node we use the repository variable `CITR_SOLO_VERSION` for our testing pipeline and that gets updated periodically with solo releases. I would recommend having something similar here.

Note `${{ vars.SOLO_VERSION }}` would need to be defined for this repository.

---

created a followup issue here for pinning Solo versions: https://github.com/hashgraph/hedera-transaction-tool/issues/1866